klasses/models.py

```diff
@@ -82,6 +85,35 @@ def payment_deadline(self):
         """
         return self.installment_set.aggregate(payment_deadline=models.Max('deadline'))['payment_deadline']
 
+    @property
+    def next_installment(self):
+        """
+        Get the next installment
+        """
+        return self.installment_set.filter(deadline__gte=datetime.datetime.now(tz=pytz.UTC)).first()
```

Comment:
There's nothing here which enforces an order of installments. Can you add an order by clause here or at the model level, and also add a test for `next_installment` which handles multiple installments?